### ISSUE TITLE
Default configuration doesn't produce graphs similar to the image anymore

### ISSUE THREAD

**Issue by averageflamethrowerguy** [danielmamay/grokking#1]
Grokking is clearly present at 25% training data, though!

**Comment by averageflamethrowerguy** [comment]
At 50%
![not_grokking](https://user-images.githubusercontent.com/58124535/211457435-15d1bb8b-7232-4b3b-8225-223933abc690.png)

At 25%
![grokking_present](https://user-images.githubusercontent.com/58124535/211457467-361d6f2e-8f96-427b-8c68-dc5689227227.png)

**Comment by bhairavmehta95** [comment]
@averageflamethrowerguy did you ever figure out what's going on here (my graphs look like yours with default HPs)

**Comment by sytelus** [comment]
I can also confirm that this code does not reproduce graph. I think author needs to put a note that this doesn't work.

![image](https://github.com/danielmamay/grokking/assets/2096835/a330939f-b33e-4491-95e3-df02ff0df8ca)

**Comment by tychovdo** [comment]
I am getting similar results as @averageflamethrowerguy when running the code provided in the repo.
It still resembles "grokking"-like behaviour (validation curves that follow training), but not quite as extreme as in the picture on the main page (generalise on validation data starts already when the training loss is not very low yet, although accuracy is good).

I wonder whether:
a) the original image was created with different hyperparam settings or another architecture?
or 
b) the difference is due to other factors (package updates, hardware, etc.)

@danielmamay, curious to hear your thoughts on this?

**Comment by bhairavmehta95** [comment]
https://github.com/KindXiaoming/Omnigrok - to everyone on the thread, been using this repo. Works out of the box.

**Comment by tychovdo** [comment]
@bhairavmehta95 Thanks for the link, I'll read the paper. However, I am looking to reproduce claims of grokking behaviour on LLMs specifically which I believe the repo you shared does not seem to provide
